packages/slo-stat-bot/src/slo-lint.ts

```diff
@@ -31,7 +33,7 @@ type Conclusion =
 
 interface ValidationResults {
   isValid: boolean;
-  errors?: ErrorObject[] | null;
+  errors?: ErrorObject[] | JSON | null;
```

Comment:
What is `JSON` in this context?  If it's a generic property bag, there are specific types I could see us using like:
- `{}` - I know this is an object, and I won't pretend to guess what the properties are
- `any` - I don't know what is going on here, and it's impossible to know.  We really avoid this if we can :)
- `{[index: string]: string}` - I know these are string based key value pairs, but that's about it

Reply:
I can create a type for displaying a message.